src/app/pages/dashboard/dashboard.component.ts

```diff
@@ -323,7 +316,10 @@ export class DashboardComponent implements OnInit, AfterViewInit, OnDestroy {
     });
 
     this.core.register({observerClass: this, eventName: 'VolumeData'}).subscribe((evt:CoreEvent) => {
-      this.setVolumeData(evt);
+      const nonBootPools = evt.data.filter(v => v.id !== 'boot-pool');
+      let clone = Object.assign({}, evt);
```

Comment:
`const`

Reply:
Not here. clone's data property is changed before it is returned

Reply:
`clone` itself is never reassigned

Reply:
kk. Changed to const